tests/sles4sap/sap_suse_cluster_connector.pm

```diff
@@ -77,11 +77,21 @@ sub run {
         my $rsc = "rsc_${rsc_type}_${instance_sid}_$instance_type$instance_id";
         wait_for_idle_cluster;
         exec_conn_cmd(binary => $binary, cmd => "lsn --res $rsc", log_file => $log_file);
-        validate_script_output("cat $log_file 2>&1", sub { m/$rsc/ });
+        # Check the "node list" contains localhost
+        my $hostname = get_required_var('HOSTNAME');
+        validate_script_output("cat $log_file | cut -d : -f 4", sub { m/$hostname/ });
+        record_info("Found $hostname in lsn output");
+        # Check the "node list" contains remote node
+        my $remote_node = script_output("crm status bynode | grep 'Node.*online' | cut -d : -f 1 | grep -v $hostname | rev | cut -d ' ' -f 1 | rev");
```

Comment:
this is short `crm status bynode | grep -Po '(?<=\* Node )(.*)(?=: online)' | grep -v $hostname`, and it is just suggestion and need verification, you can have a try if you want

Reply:
Tried  but failed, also I think your way is shorter but hard to read/understand so I would like to keep my way as it is simple:
```
> cat a
  * Node ensa-node01: online:
dddd
  * Node ensa-node02: online:
bbb
llzhao@localhost:/tmp> cat a  | grep -Po '(?<=\* Node )(.*)(?=: online$)' | grep -v node01
echo $?
1

```

Reply:
`crm status bynode | grep -Po '(?<=\* Node )(.*)(?=: online:$)' | grep -v $hostname` is correct, I lost ":". and it is OK, regex is difficult to understand

Reply:
Updated code (`"crm status bynode | grep -Po '(?<=\\* Node )(.*)(?=: online:\$)' | grep -v $hostname"`) and VR is good: https://openqa.suse.de/tests/15461873#step/sap_suse_cluster_connector/72